webcompat/templates/issue/issue-aside.html

```diff
@@ -0,0 +1,30 @@
+<aside class="wc-Issue-aside">
+  <div class="wc-Issue-information wc-Issue-information--fullWidth" id="js-Issue-aside">
+    <script type="text/template" id="aside-tmpl">
+      <header class="wc-Issue-information-header wc-Issue-information-header--<%= stateClass %>">
+        <div><%= issueState %></div>
+        <div>#<%= number %></div>
+      </header>
+      <div class="wc-Issue-information-content">
+        <h1 class="wc-Issue-title js-Issue-title">
+          <%= title %>
+        </h1>
+      </div>
+      <div class="wc-Issue-information-content">
+        <div>Opened: <%= createdAt %></div>
+        <div>
+          Repoter:
+          <span class="wc-Issue-reporter">
+            <a class="wc-Link" href="https://github.com/<%= reporter %>"><%= reporter %></a>
+          </span>
+        </div>
+        <div>Comments: <%= commentNumber %></div>
+      </div>
+    </script>
+  </div>
+  <div class="wc-Issue-information-content">
```

Comment:
@magsout it seems like backbone wants to ignore any other non-template DOM inside the templates parent. So, I had to move the `.wc-Issue-information-content .wc-Issue-labels` stuff to be a sibling. Maybe something we could tweak or fix with a wrapper? 🤔